nbviewer/formats.py

```diff
@@ -45,7 +45,7 @@ def default_formats():
             'nbconvert_template': 'slides_reveal',
             'label': 'Slides',
             'icon': 'gift',
-            'test': lambda nb, json: '"slideshow"' in json,
+            'test': lambda nb, json: 'celltoolbar' in nb.metadata and nb.metadata['celltoolbar'] == 'Slideshow',
```

Comment:
Yes that's true. The reason is, at first we have our notebook as slide, then we disable it by change all cell to - (there is no other option), but with the original condition it always catch our notebook as slide. Also, there is always a case we have text slideshow in the json itself